client/src/components/UpgradeNotification/index.js

```diff
@@ -0,0 +1,40 @@
+import { versionOutOfDate } from '../../utils/version';
+
+const initUpgradeNotification = () => {
+  const container = document.querySelector('[data-upgrade]');
+
+  if (!container) {
+    return;
+  }
+
+  /*
+  *   Expected JSON payload:
+  *   {
+  *       "version" : "1.2.3",    // Version number. Can only contain numbers and decimal point.
+  *       "url" : "https://wagtail.io"  // Absolute URL to page/file containing release notes or actual package. It's up to you.
+  *   }
+  */
+  const releasesUrl = 'https://releases.wagtail.io/latest.txt';
+  const currentVersion = container.dataset.wagtailVersion;
+
+  fetch(releasesUrl).then(response => {
```

Comment:
Another IE one, just confirming we have a fetch polyfill available.

looks like we are using whatwg-fetch, but we might want to consider an upgrade (future issue probably), we are on 2.0.3

https://github.com/github/fetch/releases

Reply:
Yeah we have a fetch polyfill, but like I mentioned in the other PR I think we should consider a more holistic polyfilling solution. Can continue discussion over there :)

Reply:
Looking at the releases of `fetch`, 3.0 doesn't bring anything we're currently using. I think it might be wise to wait until someone requires those features before updating to reduce the surface are of this PR. Sound ok @lb- ?

Reply:
@jonnyscholes  - agree 100%, that can wait until a future PR but as mentioned may not really be needed.